packages/dappmanager/src/modules/release/index.ts

```diff
@@ -5,14 +5,20 @@ import { PackageRelease, PackageRequest } from "../../types";
 import dappGet, { DappgetOptions } from "../dappGet";
 import { Apm } from "../apm";
 import { Manifest } from "@dappnode/dappnodesdk";
+import { DappGetState } from "../dappGet/types";
 
 export class ReleaseFetcher extends Apm {
   /**
    * Resolves name + version to an IPFS hash
    */
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-  async resolveReleaseName(name: string, version?: string) {
-    return resolveReleaseName(this, name, version);
+  async resolveReleaseName(
+    name: string,
+    version?: string
+  ): Promise<{
+    hash: string;
+    origin?: string | undefined;
+  }> {
+    return await resolveReleaseName(this, name, version);
```

Comment:
This await is not necessary

Reply:
Right, the issue comes when adding the eslint rule, so I return a promise in a function that is not typed to return a promise. So when I use such function the linter does not warn you to use `await`